src/app/Controllers/talking.component.ts

```diff
@@ -172,16 +188,11 @@ export class TalkingComponent implements OnInit, OnDestroy {
         if (this.content.trim().length === 0) {
             return;
         }
-        const tempMessage = new Message();
-        tempMessage.content = this.content;
-        tempMessage.sender = AppComponent.me;
-        tempMessage.sender.avatarURL = Values.fileAddress + AppComponent.me.headImgFileKey;
-        tempMessage.senderId = AppComponent.me.id;
-        tempMessage.local = true;
-        this.messages.push(tempMessage);
         this.content = AES.encrypt(this.content, this.conversation.aesKey).toString();
         this.apiService.SendMessage(this.conversation.id, this.content)
```

Comment:
When the user scrolls to the bottom of the screen, if he sends a new message, he should immediately scroll the screen to the end to display his new message. When the message is confirmed by the server, nothing should happen. (It seems that the code of the Dev branch is not the effect now)

It seems that when I send a message and the new message was not added to the list.